rally-domain/src/main/groovy/yakworks/rally/mail/services/MailMessageSender.groovy

```diff
@@ -38,8 +40,11 @@ class MailMessageSender {
         Result result
         try {
             mailTo = convertMailMessage(mailMessage)
-            // mailService.send should never throw ex and should return result
-            result = emailService.send(mailTo)
+            boolean valid = validateEmail(mailTo.to[0]) //this will throw a problem when validation fails.
+            if(valid) {
+                // mailService.send should never throw ex and should return result
+                result = emailService.send(mailTo)
+            }
         } catch(ex){
             //in convertMailMessage might throw an ex if the attachmentId is bad or not found
             result = Problem.of(ex)
```

Comment:
if validateEmail throws then we dont want to rewrap it in a problem do we?

Reply:
Fixed